apps/extension/src/routes/popup/padding-wrapper.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react';
+
+/**
+ * Wrap this around any top-level components that should have horizontal and
+ * bottom padding in the extension popup.
+ */
+export const PaddingWrapper = ({ children }: { children: ReactNode }) => {
```

Comment:
For now, this is only used in one component (the RPC settings page). Once this PR is merged, I'll make a follow-up PR to use this throughout the extension popup screens.

Reply:
it seems like a custom component is the incorrect way to do this, it should be part of a layout or other existing parent component